pkg/storage/host.go

```diff
@@ -3,17 +3,20 @@ package storage
 import (
 	"database/sql"
 	"fmt"
-
 	"github.com/kptm-tools/core-service/pkg/domain"
+	"strconv"
 )
 
 func (s *PostgreSQLStore) CreateHostsTable() error {
 	query := `create table if not exists hosts (
       id SERIAL PRIMARY KEY,
       tenant_id UUID,
       operator_id UUID,
-      value VARCHAR(2048) UNIQUE NOT NULL,
-      type VARCHAR(10) NOT NULL,
+      domain VARCHAR(2048) UNIQUE NOT NULL,
+      ip VARCHAR(15) NOT NULL,
```

Comment:
Since we may or may not have data on the IP-Domain (we could have one or the other) I suggest removing the NOT NULL for now.

Reply:
Ok, I removed it